glossary.yml

```
@@ -2875,14 +2875,20 @@
 - slug: harmonic_mean
 
 
+- slug: hardware
+  en:
+    term: "hardware"
+    acronym: "HW"
+    def: >
+      Any physical component of a computer system. Hardware can be internal, such as CPU, memory, and graphic cards, or external, such as monitors and keyboards. The hardware operates in conjunction with software to produce a functioning computer system.
```

```suggestion
      Any physical component of a computer system. Hardware can be internal, such as CPU, memory,
      and graphics cards; or external, such as monitors and keyboards. Hardware operates in conjunction
      with software to produce a functioning computer system.
```

Keep lines from being too wide.